powerhome/playbook
# [Play-568] typeahead error state

#### Screens

<img width="765" alt="Screenshot 2023-02-02 at 9 21 12 AM" src="https://user-images.githubusercontent.com/73710701/216351288-38ea650e-9d18-4e87-8eb9-09ad64d5ad73.png">


#### Breaking Changes

NO

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-568)

#### How to test this

Add error state to other typeahead examples and make sure it functions as it should

#### Checklist:
